packages/webapp/public/locales/pt/translation.json

```diff
@@ -1791,19 +1792,18 @@
     },
     "TEMPERATURE_READINGS_OF_SENSOR": {
       "TITLE": "Temperatura do solo",
-      "SUBTITLE": "Temperatura ambiente máxima e mínima de hoje: {{high}}{{tempUnit}} / {{low}}{{tempUnit}}",
-      "Y_AXIS_LABEL": "Temperatura em {{tempUnit}}",
+      "SUBTITLE": "Temperatura ambiente máxima e mínima de hoje: {{high}}{{units}} / {{low}}{{units}}",
+      "Y_AXIS_LABEL": "Temperatura em {{units}}",
       "WEATHER_STATION": "Estação meteorológica: {{weatherStationLocation}}",
-      "NO_DATA": "(sem dados))",
-      "AMBIENT_TEMPERATURE_FOR": "Temperatura ambiente para",
-      "LAST_UPDATED_TEMPERATURE_READINGS": "Última atualização {{latestReadingUpdate}}"
+      "AMBIENT_TEMPERATURE_FOR": "Temperatura ambiente para"
     },
     "SOIL_WATER_POTENTIAL_READINGS_OF_SENSOR": {
       "TITLE": "Potencial hídrico do solo",
-      "SUBTITLE": "MISSING",
-      "Y_AXIS_LABEL": "Potencial hídrico do solo em {{soilWaterPotentialUnit}}",
-      "NO_DATA": "(sem dados))",
-      "LAST_UPDATED_TEMPERATURE_READINGS": "Última atualização {{latestReadingUpdate}}"
+      "Y_AXIS_LABEL": "Potencial hídrico do solo em {{units}}"
```

Comment:
So this is one of the phrases that was translated in two different ways for the sensor reading types, and the version that Isabela verified is potencial_de_água_do_solo -- and there was even a third translation that was not grammatical and removed!

![Screenshot 2023-05-12 at 2 37 37 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/6a059dc1-321b-4e2b-8720-0133a6c29f09)

Do you think we should keep these consistent across the CSV and the chart view? ChatGTP says that hídrico is more technical so I wonder if the CSV should be updated to use this too.

And then if we are going to keep the phrasing consistent in these different places, I wonder if we should pre-populate here (rather than using the MISSING tags) with the validation translations? What do you think?

Reply:
@kathyavini 

I think it is a good point. My comments for this might be:
- Maybe where it is critical if used in multiple places for validation we the developers should likely add some sort of comment or warning or separate file highlighting the implications of changing translations
- where it is only "inconsistent" maybe a process suggestion for translation PRs could be proposed where a search for "like" translations is done and an attempt at consistency is made

Reply:
Hmmm! Very good point that in non-validation contexts it's not critical.

I don't love the idea of putting more work on the translators by having them refer to a process doc if we could just enforce it by doing the translations ourselves (for devs I don't mind an external doc!)... but in any case, new issue, new ticket 😁 I'll bring it in Standup to see how consistent we want these non-critical contexts.

Reply:
Process doc is always good even for trivial things!.... but yah translation file is also in need of some de-duping and dev strategy too.